gridstatus/pjm.py

```diff
@@ -941,6 +941,64 @@ def extract_short_name(row):
 
         return data
 
+    @support_date_range(frequency=None)
+    def get_it_sced_lmp_5_min(self, date, end=None, verbose=False):
+        """Get 5 minute LMPs from the Integrated Forward Market (IFM)"""
+
+        if date == "latest":
+            return self.get_it_sced_lmp_5_min("today", verbose=verbose)
+
+        params = {
+            "fields": (
+                "case_approval_datetime_utc,datetime_beginning_utc,itsced_lmp,marginal_congestion,marginal_loss,pnode_id,pnode_name"  # noqa: E501
+            ),
+        }
+
+        df = self._get_pjm_json(
+            "five_min_itsced_lmps",
+            start=date,
+            end=end,
+            params=params,
+            verbose=verbose,
+            interval_duration_min=5,
+        )
+
+        df.columns = df.columns.map(lambda x: x.replace("_", " ").title())
+
+        df = df.rename(
+            columns={
+                "Case Approval Datetime Utc": "Case Approval Time",
+                "Itsced Lmp": "LMP",
+                "Pnode Id": "Pnode ID",
+                "Marginal Congestion": "Congestion",
+                "Marginal Loss": "Loss",
+            },
+        )
+
+        # LMP = Energy + Congestion + Loss so Energy = LMP - Congestion - Loss
+        df["Energy"] = df["LMP"] - df["Congestion"] - df["Loss"]
+
+        df = df[
+            [
+                "Interval Start",
+                "Interval End",
+                "Case Approval Time",
+                "Pnode ID",
```

Comment:
We probably want to convert this to a `location` like we do with the other PJM datasets. I'll work on that.

Reply:
Updated to add a `location` as we do with the other PJM LMP datasets.